### ISSUE TITLE
libnvToolsExt.so.1 is not a symbolic link

### ISSUE THREAD

**Issue by linsalrob** [nanoporetech/dorado#390]
When installing `dorado-0.3.4-linux-x64` I received the error: `libnvToolsExt.so.1 is not a symbolic link`

## Steps to reproduce
[note you will need to do this as `sudo`, but I omitted that to avoid people just copying and pasting without thinking]

 - Download and unpack `https://cdn.oxfordnanoportal.com/software/analysis/dorado-0.3.4-linux-x64.tar.gz`
 - make library location: `mkdir /usr/lib/dorado-0.3.4-linux-x64 && cp -r lib/ /usr/lib/dorado-0.3.4-linux-x64/`
 - add the path to ldconfig: `echo "/usr/lib/dorado-0.3.4-linux-x64/lib" > /etc/ld.so.conf.d/dorado.conf`
 - run `ldconfig`: 

```
$ sudo ldconfig
/sbin/ldconfig.real: /usr/lib/dorado-0.3.4-linux-x64/lib/libnvToolsExt.so.1 is not a symbolic link
```

## Expected behaviour
`ldconfig` should return 0

## Steps to fix the error

Remove both `libnvToolsExt.so.1` and `libnvToolsExt.so.1.0.0` and set them as symlinks to `libnvToolsExt.so`:

```
rm -f libnvToolsExt.so.1 libnvToolsExt.so.1.0.0
ln -s libnvToolsExt.so libnvToolsExt.so.1 
ln -s libnvToolsExt.so libnvToolsExt.so.1.0.0
```

**Comment by susie-ont** [comment]
Hi @linsalrob - thanks for raising this! The team is looking into it and will work on getting a fix.

**Comment by tijyojwad** [comment]
Hi @linsalrob - this should be fixed in [v0.4.3](https://github.com/nanoporetech/dorado#installation) released yesterday. Please let us know if you continue to run into problems.
